### PR TITLE
Fixing some URLs

### PR DESCRIPTION
**Pull request by AndersonTorres**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/sakura/default.nix [hunk 1]
```diff
@@ -1,27 +1,37 @@
...
+{ stdenv
+, fetchurl
+, cmake
+, pkgconfig
+, gtk3
+, perl
+, vte
+, pcre2
+, glib 
+, makeWrapper }:
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/sakura/default.nix:10]: Please try avoid unnecessary formatting changes. It makes seeing the real changes harder and backports are not as easy.

Comment (worldofpeace) [reply pkgs/applications/misc/sakura/default.nix:10]: There's actually hide whitespace changes on github and with git also.

Comment (worldofpeace) [inline pkgs/applications/misc/sakura/default.nix:10]:
```suggestion
, makeWrapper 
}:
```

Comment (SuperSandro2000) [reply pkgs/applications/misc/sakura/default.nix:10]: Doesn't seem to work here.

File: pkgs/applications/misc/sakura/default.nix [hunk 2]
```diff
@@ -32,5 +44,8 @@ stdenv.mkDerivation rec {
       terminals in one window and adds a contextual menu with some basic
       options. No more no less.
     '';
-  };
+    license     = licenses.gpl2;
```
Comment (jtojnar) [inline pkgs/applications/misc/sakura/default.nix:47]: `gpl3` is unclear, use `gpl3Plus` or `gpl3Only`. See https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206 for more details.

File: pkgs/applications/video/bombono/default.nix [hunk 3]
```diff
@@ -1,20 +1,38 @@
...
+, enca
+, wrapGAppsHook
+}:
+
+let
+  fetchPatchFromAur = {name, sha256}:
+    fetchpatch {
+      inherit name sha256;
+      url = "https://aur.archlinux.org/cgit/aur.git/plain/${name}?h=e6cc6bc80c672aaa1a2260abfe8823da299a192c";
+      };
```
Comment (jtojnar) [inline pkgs/applications/video/bombono/default.nix:25]:
```suggestion
    };
```

File: pkgs/applications/video/bombono/default.nix [hunk 4]
```diff
@@ -38,17 +56,17 @@ stdenv.mkDerivation rec {
 
   buildInputs = [
     gtk2 gtkmm2 mjpegtools libdvdread dvdauthor boost dvdplusrwtools
-    libxmlxx ffmpeg_3 enca
-    ];
+    libxmlxx ffmpeg_3 enca
```
Comment (jtojnar) [inline pkgs/applications/video/bombono/default.nix:59]:
```suggestion
    libxmlxx ffmpeg_3 enca
```

File: pkgs/development/libraries/babl/default.nix [hunk 5]
```diff
@@ -33,9 +33,9 @@ stdenv.mkDerivation rec {
 
   meta = with stdenv.lib; {
     description = "Image pixel format conversion library";
-    homepage = "http://gegl.org/babl/";
-    license = licenses.gpl3;
-    maintainers = with stdenv.lib.maintainers; [ jtojnar ];
+    homepage = "https://gegl.org/babl/";
+    license = licenses.gpl3Plus;
```
Comment (jtojnar) [inline pkgs/development/libraries/babl/default.nix:37]: Looks like it is actually `lgpl3Plus`: https://gegl.org/babl/index.html#Copyright

File: pkgs/development/libraries/gegl/4.0.nix [hunk 6]
```diff
@@ -120,8 +120,8 @@ stdenv.mkDerivation rec {
 
   meta = with stdenv.lib; {
     description = "Graph-based image processing framework";
-    homepage = "http://www.gegl.org";
-    license = licenses.gpl3;
+    homepage = "https://www.gegl.org";
+    license = licenses.gpl3Plus;
```
Comment (jtojnar) [inline pkgs/development/libraries/gegl/4.0.nix:124]: Looks like it is actually `lgpl3Plus`: https://gitlab.gnome.org/GNOME/gegl/-/commit/6609e9b8e1a5f8fd8d06de6109f45a8e43c4f6a3

File: pkgs/tools/video/atomicparsley/default.nix [hunk 7]
```diff
@@ -1,39 +1,29 @@
...
 
-  configureFlags = stdenv.lib.optionals (stdenv.hostPlatform != stdenv.buildPlatform) [
-    # AC_FUNC_MALLOC is broken on cross builds.
-    "ac_cv_func_malloc_0_nonnull=yes"
-    "ac_cv_func_realloc_0_nonnull=yes"
-  ];
-
-  installPhase = "install -D AtomicParsley $out/bin/AtomicParsley";
+  installPhase = ''
+    install -D AtomicParsley $out/bin/AtomicParsley
```
Comment (jtojnar) [inline pkgs/tools/video/atomicparsley/default.nix:20]: When overriding `installPhase`, you should also provide `runHook preInstall` and `runHook postInstall`.

File: pkgs/applications/misc/sakura/default.nix (hunk 2 above)
Comment (AndersonTorres) [reply pkgs/applications/misc/sakura/default.nix:47]: It is GPL2Only

File: pkgs/tools/video/atomicparsley/default.nix (hunk 7 above)
Comment (AndersonTorres) [reply pkgs/tools/video/atomicparsley/default.nix:21]: Is this some new directive? I was not aware of it until now...
